app-engine/java/gradle/src/com/google/cloud/tools/intellij/appengine/java/gradle/DefaultAppEngineGradleModel.java

```diff
@@ -23,10 +23,13 @@
 
   private boolean hasAppEngineGradlePlugin;
   private String gradleBuildDir;
+  private String gradleModuleDir;
```

Comment:
Since these fields are effectively read-only, should we also mark them final?

Reply:
done